runbooks/runbook.md

```diff
@@ -0,0 +1,118 @@
+# UPP - Content Preview
+
+Content Preview accesses unpublished content and previews it before publishing.
```

Comment:
This information is not completely true. It is not a preliminary check before publishing. The services provides an option to preview content (mainly published from Methode) closely to what it is going to look like on ft.com. Please edit it a bit.
You can shorten and use the first sentence from the Architecture part/ the README description.